src/test/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRendererUnitTest.java

```diff
@@ -1163,8 +1163,9 @@ public void testWrite(final List<VariantContext> variants, final List<List<Funco
             }
         }
 
+        // Make sure our files are as we expect them to be:
         try {
-            IntegrationTestSpec.assertEqualTextFiles(outFile, expectedFile, "#");
+            IntegrationTestSpec.assertEqualTextFiles(outFile, expectedFile, MafOutputRendererConstants.COMMENT_STRING, false);
```

Comment:
Add a comment about not ignoring trailing whitespace here.

Reply:
👍